magic-8-ball/06-deploy-fermyon-cloud.md

```diff
@@ -29,3 +29,4 @@ In this section you learned how to:
 
 - Go back to [05 - Persisting Magic 8 Ball Responses](05-spin-kv.md) if you still have questions on previous section
 - Otherwise, proceed to [07 - Storing data in an external database](07-external-db.md)
+- To learn how to deploy this Spin application to Kubernetes using SpinKube, check out this step in our [Wasm and Containers workshop]([url](https://github.com/fermyon/workshops/blob/main/wasm-and-containers/workshop/04-kubernetes-and-spinkube.md)). **Note**: This workshop has its own pre-requisites.
```

Comment:
Does the `[url]` bit work as intended?  When previewing this change, the link doesn't appear to quite work for me.

```suggestion
- To learn how to deploy this Spin application to Kubernetes using SpinKube, check out this step in our [Wasm and Containers workshop](https://github.com/fermyon/workshops/blob/main/wasm-and-containers/workshop/04-kubernetes-and-spinkube.md). **Note**: This workshop has its own pre-requisites.
```